static/js/containers/pages/profile/EditProfilePage.js

```diff
@@ -82,13 +84,28 @@ export class EditProfilePage extends React.Component<Props, State> {
 
     try {
       const {
+        /* eslint-disable camelcase */
         body: { errors },
       }: { body: Object } = await editProfile(payload);
 
-      if (errors && errors.length > 0) {
-        setErrors({
-          email: errors[0],
+      if (errors) {
```

Comment:
Can we utilize the functionality of the Validation exception from the backend instead of making this code change in the frontend? But in reality are we sending multiple error messages from the backend?

Reply:
> Can we utilize the functionality of the Validation exception from the backend instead of making this code change in the frontend?
Is that change break anything with existing functionality?

I'm not sure if you have this question due to ` email: errors[0],`, I tried to check email but did not get it's usage in on edit profile

Reply:
I mean, when we return ValidationError from the backend would that become part of the field_errors?

Currently, We return a Validation Error in case legal address validation, But that doesn't get displayed on the frontend, That however, stops the registration process keeping the user on the same page. Take a look at [this](https://github.com/mitodl/mitxpro/blob/master/users/serializers.py#L51-L77) for reference. And I see that you are also throwing a validation error message. So, we don't need these frontend changes If that's still not going to be shown on the fields.